src/Symfony/Component/Validator/Constraints/AtLeastOneOf.php

```diff
@@ -26,7 +26,7 @@ class AtLeastOneOf extends Composite
     ];
 
     public $constraints = [];
-    public $message = 'This value should satisfy at least one of the following constraints:';
+    public $message = 'This value should satisfy at least one of the following constraints: {{ child_messages }}';
```

Comment:
As of today, translations are the same in all maintained branches, which simplifies things a lot. That would break this pattern as this message would be different depending on the version. I would love to find another way to fix this issue.

Reply:
Alright, thank you for your feedback. I'll try to get back with another solution! :+1:

Reply:
can't we concatenate the other messages after translating the current message?
that would allow not changing the translation tables

Reply:
Oh I forgot about this one. I'll have a look at it soon and give it another try with your additional info!